src/app/[locale]/(home)/cart/_components/cart-item.tsx

```
@@ -3,41 +3,103 @@
 import Image from "next/image";
 import { Card, CardContent } from "@/components/ui/card";
 import { Button } from "@/components/ui/button";
-import { Minus, Plus, Trash2, Star } from "lucide-react";
+import { Minus, Plus, Trash2, Star, Loader2 } from "lucide-react";
 import Link from "next/link";
-import { useState } from "react";
+import { useState, useRef, ChangeEvent } from "react";
 import { Input } from "@components/ui/input";
+import { toast } from "sonner";
+import { useRouter } from "@i18n/navigation";
```

ESLint

[error] 'useRouter' is defined but never used.

(@typescript-eslint/no-unused-vars)